### ISSUE TITLE
Active datasource and scema buttons missing sql editor

### ISSUE THREAD

**Issue by bsienn** [serge-rider/dbeaver#1387]
OS: Ubuntu 16.04 64bit.

Reproduce:
1) Open Dbeaver
2) Close default sql editor
3) Open new sql eitor

![untitled](https://cloud.githubusercontent.com/assets/3149876/23541694/cf366998-000a-11e7-8f01-73db765e234d.png)

**Comment by serge-rider** [comment]
They are there but they were narrowed (for some reason).
* Move toolbars and then restart.
* Exit DBeaver and delete ~/.dbeaver4/.metadata/.plugins/org.eclipse.e4.workbench/workbench.xmi . Then start DBeaver.

**Comment by bsienn** [comment]
yep, its there and shrieked.
Rearranging does bring them back temporally but closing and reopening editor is having the same effect.
Deleting workbench.xml and restarting didn't solved the issue.

**Comment by serge-rider** [comment]
I saw such behavior in 4.0 and it was fixed. Now I can't reproduce it on my Ubuntu.
Is there anything special in your Windows manager configuration?

**Comment by bsienn** [comment]
Now that you mention. I just found out whats the issue.
Its working fine now.

**Details:**
When I first started Dbeaver it made 1 icon on sidebar (Dbeaver-CE).
On first run. it ran the app and started showing 2 icons in sidebar
One for app launcher and one for the actual running app (Dbeaver-general).
I didn't liked it and without second thoughts I marked and pinned the running app Icon in sidebar.

This was my 1st encouner with Dbeaver on ubunto, so I had no idea.

Starting the app with Dbeaver-CE  launcher is working fine.
Starting the app with Dbeaver-general causes the issue.
